tests/external_estimators/ext_sdfq.sage

```diff
@@ -0,0 +1,118 @@
+load("tests/external_estimators/helpers/attack_cost.sage")
+load("tests/external_estimators/helpers/cost.sage")
+
+from math import comb as binomial, log2
+from itertools import chain
+
+import sage.all
+
+
+def lee_brickell_correction(k: int) -> float:
+    """
+    Calculate the correction term for the Lee-Brickell procedure.
+
+    Args:
+        k (int): The dimension of the code.
+
+    Returns:
+        float: The calculated correction term.
+
+    Notes:
+        See SDFqAlgorithms/leebrickell.py line 98/99
+    """
+    return log2(k) * 2 - log2(binomial(k, 2))
+
+
+def ext_lee_brickell():
+    """Generate expected complexities for Lee-Brickell SDFq problems.
+
+    This function calculates the expected complexities for a predefined set of
+    Lee-Brickell SDFq problem parameters.
+
+    Returns:
+        list of tuple: Each tuple contains:
+            - tuple: Input parameters (n, k, w, q)
+            - float: Corresponding expected complexity
+    """
+
+    inputs = [(256r, 128r, 64r, 251r), (961r, 771r, 48r, 31r)]
```

Comment:
raw numbers? is this the problem javi explained us, that sage serialize them as `sage integers`?

Reply:
It's the other way around, in fact 😅. Putting them as raw numbers is what avoids the preparser type casting.